GPU/Common/VertexShaderGenerator.cpp

```diff
@@ -921,9 +922,19 @@ bool GenerateVertexShader(const VShaderID &id, char *buffer, const ShaderLanguag
 
 		// Final view and projection transforms.
 		if (gstate_c.Supports(GPU_ROUND_DEPTH_TO_16BIT)) {
-			WRITE(p, "  vec4 outPos = depthRoundZVP(mul(u_proj%s, viewPos));\n", matrixPostfix.c_str());
+			if (IsVRBuild()) {
+				WRITE(p, "  vec4 outPos = depthRoundZVP(mul(u_proj_lens%s, viewPos));\n", matrixPostfix.c_str());
+				WRITE(p, "  vec4 orgPos = depthRoundZVP(mul(u_proj, viewPos));\n", matrixPostfix.c_str());
```

Comment:
There's no % in these line (or the one below) - should there be after u_proj?  Or if not, shouldn't have the arg.

-[Unknown]